modules/apps/list-type/list-type-service/src/main/java/com/liferay/list/type/service/impl/ListTypeDefinitionLocalServiceImpl.java

```diff
@@ -127,6 +129,7 @@ public ListTypeDefinition deleteListTypeDefinition(
 	@Override
 	public ListTypeDefinition updateListTypeDefinition(
 			String externalReferenceCode, long listTypeDefinitionId,
+			long userId, List<ListTypeEntry> listTypeEntries,
```

Comment:
I think we should order the parameters by following the order from service.xml. Since ListTypeDefinition does not contain the listTypeEntries, so I believe that this parameter should come at last.